### ISSUE TITLE
Two tabs are being opened

### ISSUE THREAD

**Issue by rampaa** [stoically/temporary-containers#323]
- Temporary Containers Version: 1.4beta2
- Firefox Version: 69.0.3 64 bit
- Operating System + Version: Windows 10
- Other installed Add-ons + Version: Containerise 3.7.0

### Actual behavior
When I middle-click to a link that directs me to a website that I have a permanent container for, two tabs are opened instead of one. One of which is a blank temporary container.

### Expected behavior
Only one tab should be opened and that tab (a) should not be blank (b) should be within its specified container.

### Steps to reproduce
1. Install Containerise
2. Enable the following option: Containerise>Options>Keep old tabs 
2. Add the rule `github.com , Github` to Containerise.
3. Make sure you don't have a rule for google.com in Containerise.
4. Search Github in google.com. 
5. Middle-click to the first result.

Log file: [Log.zip](https://github.com/stoically/temporary-containers/files/3827949/Log.zip)

Note: With Temporary Containers v1.3, the issue is arguably worse. In v1.3, when the aforementioned steps are taken, github.com won't be opened in the Github container at all. There will only be one tab opened and tab will be all blank save for its address bar.

**Comment by gingerlime** [comment]
I'm having a similar issue but I'm not using containerise (I have it installed, but disabled). I do have conex though, which is similar?

**Comment by gingerlime** [comment]
Looks related https://github.com/kesselborn/conex/issues/307

**Comment by crssi** [comment]
Conex and Containerise are not similar.

**Comment by gingerlime** [comment]
Yes, you're right. I'm sorry. I guess they just both deal with containers and have a somewhat similar icon :)

![Screenshot 2020-03-04 at 07 49 37](https://user-images.githubusercontent.com/524274/75852659-cb23af80-5dec-11ea-89ed-96599bad80af.png)
![Screenshot 2020-03-04 at 07 49 51](https://user-images.githubusercontent.com/524274/75852662-cbbc4600-5dec-11ea-977b-b7ff3fd1efda.png)
